.taskcluster.yml

```diff
@@ -0,0 +1,313 @@
+                              else:
+                                  '${ownTaskId}'  # same as taskId; this is how automation identifies a decision task
+                          schedulerId: '${trustDomain}-level-${level}'
+                          created: {$fromNow: ''}
+                          deadline: {$fromNow: '1 day'}
+                          expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
+                          metadata:
+                              $merge:
+                                  - owner: "${ownerEmail}"
+                                    source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
+                                  - $if: 'tasks_for in ["github-push", "github-pull-request", "github-release"]'
+                                    then:
+                                        name: "Decision Task"
+                                        description: 'The task that creates all of the other tasks in the task graph'
+                                    else:
+                                        $if: 'tasks_for == "action"'
+                                        then:
+                                            name: "Action: ${action.title}"
+                                            description: '${action.description}'
+                                        else:
+                                            name: "Decision Task for cron job ${cron.job_name}"
+                                            description: 'Created by a [cron task](https://focus-ci-tc.services.mozilla.com/tasks/${cron.task_id})'
+                          provisionerId: "mobile-${level}"
+                          workerType: "decision"
+                          tags:
+                              $if: 'tasks_for in ["github-push", "github-pull-request"]'
+                              then:
+                                  kind: decision-task
+                              else:
+                                  $if: 'tasks_for == "action"'
+                                  then:
+                                      kind: 'action-callback'
+                                  else:
+                                      $if: 'tasks_for == "cron"'
+                                      then:
+                                          kind: cron-task
+                          routes:
+                              $flattenDeep:
+                                  - checks
+                                  - $if: 'level == "3"'
+                                    then:
+                                        - tc-treeherder.v2.${project}.${head_sha}
+                                        - $if: 'tasks_for == "github-push"'
+                                          then:
+                                              - index.mobile.v2.${project}.branch.${short_head_branch}.latest.taskgraph.decision
```

Comment:
removing project as in https://github.com/mozilla-mobile/firefox-ios/pull/6302/files
and moving v2 nexto to mobile as in https://github.com/mozilla-mobile/firefox-ios/pull/6303/files